chapters/errors.Rmd

```diff
@@ -751,8 +757,8 @@ can save us and our users a lot of needless work.
 
 ## Exercises {#errors-exercises}
 
-This chapter suggested several edits to `collate.py`,
-such that the script now reads:
+This chapter suggested several edits to `collate.py`.
+We'll proceed with the following basic framework for the script:
```

Comment:
"We'll use" rather than "We'll proceed with" ?

Reply:
"basic framework" is also a bit of a strange expression... I'd probably stick with "the script now reads as follows"

Reply:
Changed to "If our script now reads as follows:", and then after the code there's a sentence that says the following exercises will ask you to make further edits.."